test/integration/service/loadbalancer_test.go

```diff
@@ -23,14 +23,19 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/client-go/informers"
 	clientset "k8s.io/client-go/kubernetes"
 	restclient "k8s.io/client-go/rest"
+	servicecontroller "k8s.io/cloud-provider/controllers/service"
+	fakecloud "k8s.io/cloud-provider/fake"
 	featuregatetesting "k8s.io/component-base/featuregate/testing"
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/test/integration/framework"
 	utilpointer "k8s.io/utils/pointer"
 )
 
+const clusterName = "test-cluster"
```

Comment:
I think we can remove this constant and just put it inline where it's used, since it's only used in one place